src/main/java/frc/robot/subsystems/pivot/PivotIOTalonFX.java

```diff
@@ -219,6 +222,18 @@ public double getPivotTarget() {
     return pivotTargetAngle;
   }
 
+  /**
+   * sets the pivot angle(rotations) shooter
+   *
+   * @param angle the desired angle in rotations
+   */
+  @Override
```

Comment:
usually when writing docstrings in java, if you have @override, you should put the docstring for the super class' method. @Ishan1522 I'm not goofing here right?

Reply:
yes, it needs to be documented from the interface.